tutorials/multicore/mt303_fillHistosAutobin.C

```diff
@@ -0,0 +1,135 @@
+#include "TF1.h"
+#include "TH1D.h"
+#include "TMath.h"
+#include "TF1.h"
+#include "TCanvas.h"
+#include "TRandom3.h"
+#include "TStatistic.h"
+#include "TFile.h"
+#include "TStyle.h"
+
+TF1 *gam = new TF1("gam", "1/(1+0.1*x*0.1*x)", -100., 100.);
```

Comment:
Do those function (pointer) really need to be global?

Reply:
No. This was a working macro, not really intended to make it to the end ... at least without proper revision.